### DISCUSSION TITLE
New game: "For how much?"

### DISCUSSION THREAD

**Discussion by AtwerJ** [Glazelf/NinigiBot#48] [Ideas]
It's a 1v1, PvP game. Basically the **challenger (P1)** says "For how much you [do x thing or whatever]?" and the **challenged (P2)** gives a number, like "For 5." Both of them will say a number (at the same time IRL), if the numbers you gave are different, the challenger loses. But if the numbers are the same, the challenged loses!
**P1:** "For how much?"
**P2:** "For 6"
**P1:** 6!
**P2:** 3!

With Ninigi, it'd work like a command, for example **"?challenge @user [amountofmoney]"**, the other user has the choice to _accept or decline the challenge_. Ninigi then asks the other user (P2) _what number does he want to go with_, and once the number is decided, the players have to _DM Ninigi the numbers they chose_. Once both players did that, _Ninigi announces the numbers picked by both players and the winner_.
Edit: Also, I think there should be a limit for the bot, to how low the number can be like 3.

**Comment by GIVEMEAUSERNAMETHEFUCK** [comment]
I personally think this is a suboptimal idea, while you may disagree my intellectual viewpoint is that the contributors should first focus on #47 
But alas this is but my personal point of view.

**Comment by Glazelf** [comment]
Closing this since gambling rules changed.
